students/rmart300/lesson09/donor.py

```diff
@@ -0,0 +1,18 @@
+
+
+class Donor(object):
```

Comment:
In Python 3 it's not necessary to explicitly inherit from `object` (though all classes do by default). So this can be `class Donor:` without the parens.